gradle/pmd.gradle

```diff
@@ -16,9 +16,10 @@
  */
 apply plugin: 'pmd'
 pmd {
-  sourceSets = [sourceSets.main]
+  sourceSets = []
```

Comment:
Why are we not running on the main source set anymore?

Reply:
Conversely, does this trivialize the task in the `execute_build.sh`?

Reply:
This means pmdMain does not run as part of check. Because we are manually configuring the pmdMain task (a few lines below this) the task is still there.